src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/Offender.kt

```diff
@@ -56,6 +56,9 @@ data class Offender(
   @NotNull
   val lastUpdatedTimestamp: LocalDateTime = LocalDateTime.now(),
 ) {
+
+  fun currentAssessment(): Assessment = this.assessments.first { it.status == AssessmentStatus.NOT_STARTED || it.status === AssessmentStatus.OPTED_OUT }
```

Comment:
I've promoted this to the entity so we can reuse it easily

Reply:
think I did that as well in my last PR as I needed it in two services, but not showing a conflict?